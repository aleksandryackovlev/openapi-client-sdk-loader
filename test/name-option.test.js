import {
  compile,
  execute,
  getCompiler,
  getErrors,
  getWarnings,
  readAsset,
} from './helpers';

describe('"name" option', () => {
  it('should work with "Boolean" value equal "true"', async () => {
    const compiler = getCompiler('simple.js', {
      name: true,
    });
    const stats = await compile(compiler);

    expect(
      execute(readAsset('main.bundle.js', compiler, stats))
    ).toMatchSnapshot('result');
    expect(getErrors(stats)).toMatchSnapshot('errors');
    expect(getWarnings(stats)).toMatchSnapshot('warnings');
  });
});
