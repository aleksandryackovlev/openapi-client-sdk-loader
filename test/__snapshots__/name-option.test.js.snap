// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`"name" option should work with "Boolean" value equal "true": errors 1`] = `Array []`;

exports[`"name" option should work with "Boolean" value equal "true": result 1`] = `"foo"`;

exports[`"name" option should work with "Boolean" value equal "true": warnings 1`] = `Array []`;
